src/components/shared/Select/index.tsx

```diff
@@ -166,6 +167,19 @@ const getThemeProp = ({ theme, comp, prop }: ThemePropParams): string => {
   return themeStylePropCollection[theme][comp][prop];
 };
 
+const SelectContainer = styled.View`
+  z-index: 1;
+  margin-bottom: 20px;
+`;
+const Title = styled.Text<ThemeType>`
+  font-size: 14px;
+  color: ${(props): string =>
+    getThemeProp({
+      theme: props.theme,
+      comp: CompEnum.text,
+      prop: StylePropEnum.fc,
+    })};
+`;
```

Comment:
Thanks for creating a separate component in my opinion. However, the Title component you created looks different from the default title shown in issue #60. I'd appreciate it if you could change the font size, color, and position to make it similar to #60.
 <img src="https://user-images.githubusercontent.com/33364619/70374854-d32d9980-193a-11ea-931a-320825f469aa.png" width="300" alt="Select Structure" />

Reply:
Yup!
I've modifed the size and the position of title component.
You can see the difference through the screenshot on the top of this page.